### DISCUSSION TITLE
I can’t access multiple accounts at once

### DISCUSSION THREAD

**Discussion by PuRu5heR** [thedemons/opentele#101] [Q&A] [answered]
Good day!
I want to thank you for a very useful library.

I have problems with accessing multiple accounts at once. My task is to update session of telegram accounts. But when I go through a loop through several folders with tdata for different users, on the second account I get the following error: 
sqlite3.OperationalError: database is locked
That comes from 
client = await tdesk.ToTelethon(session="telethon.session", flag=UseCurrentSession)
I really dont know what to do, help me please

**Comment by thedemons** [comment, marked as answer]
You probably use the same file "telethon.session" while the old telethon instance is still using it, either close the old instance or use a different file name for each one.

**Reply by PuRu5heR** [reply]
I thought about it, i was trying to find how to close it, but maybe different naming will be better solution. 
Thank you so much!
